modules/social_features/social_core/social_core.module

```diff
@@ -865,6 +896,22 @@ function social_core_form_node_form_alter(&$form, FormStateInterface $form_state
           $form['author_info']['created'] = $form['created'];
           unset($form['created']);
         }
+
+        $has_option_fields = FALSE;
+
+        foreach (['promote', 'sticky'] as $field) {
+          if (isset($form[$field])) {
+            $form['options'][$field] = $form[$field];
+            unset($form[$field]);
+            $has_option_fields = TRUE;
+          }
+        }
+
+        if ($has_option_fields) {
```

Comment:
@OH17L If some developers want to use Promoted and Sticky options, and enable it by purpose, will it be displayed in form? Please add a comment on what happens here.

Reply:
Done.